### ISSUE TITLE
Is it work for you? 

### ISSUE THREAD

**Issue by felipe-moz** [borigas/NotificationServiceExtensionRepro#1]
Hi, I would like to know if you were able to make it work for you? I have the same scenario that yours but my ios notification extension doesn't work, I've replicated my sample project in swift and it works fine but in MAUI don't. I really appreciate any help. Thanks

**Comment by borigas** [comment]
Nope, this project is meant to prove that it doesn't work in an easily reproducible example. The issue got bounced between MSFT teams a [few](https://github.com/dotnet/maui/issues/22586) [times](https://github.com/xamarin/xamarin-macios/issues/20918), but [this is the current issue tracking the problem](https://github.com/microsoft/vscode-dotnettools/issues/1386). There's also a [developer community feedback issue for it](https://developercommunity.visualstudio.com/t/iOS-Notification-Service-Extension-Deplo/10665806?viewtype=all). Please give them an upvote to help bring attention to the issue.

**Comment by felipe-moz** [comment]
I was able to deploy a sample project using vscode. If it helps somehow let me know


https://github.com/user-attachments/assets/27384d75-2ed6-4fd1-99d5-c2383c81b48a
